src/trackers/TVC.py

```
@@ -251,97 +303,150 @@ async def upload(self, meta, disctype):
+                    meta['episodes'] = []
+                    for ep in season_info.get('episodes', []):
+                        code = f"S{str(ep.get('season_number', 0)).zfill(2)}E{str(ep.get('episode_number', 0)).zfill(2)}"
+                        meta['episodes'].append({
+                            "code": code,
+                            "title": (ep.get("name") or "").strip(),
+                            "airdate": ep.get("air_date") or "",
+                            "overview": (ep.get("overview") or "").strip()
+                        })
+                    if hasattr(tv, 'first_air_date'):
+                        meta['first_air_date'] = tv.first_air_date
+            except Exception:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)